lib/algos/list/delimiter.js

```diff
@@ -61,8 +61,8 @@ class Delimiter extends Extension {
      *                                                   alphabetically ordered
      *                                                   or not
      */
-    constructor(parameters) {
-        super(parameters);
+    constructor(parameters, logger) {
```

Comment:
As described in commit number 2, I had to add this for tests compatibility.
As the loggers are actually provided by cloudserver & metadata, there should be no issue here, and we should have the logger to log in the base Extension's `filterContent()` method.